google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -384,6 +395,10 @@ public static QueryParameterValue struct(Map<String, QueryParameterValue> struct
         break;
       case STRING:
         return value.toString();
+      case JSON:
+        if (value instanceof String) {
```

Comment:
If JSONObject support were added, I'm assuming that this too would need to be updated

Reply:
Yes, updating this to parse JSON string also.